src/decision/include/LidarDecision.h

```diff
@@ -140,7 +154,7 @@ class LidarDecision {
      * @return a vector of all obstacles
      */
     static std::vector<LidarObstacle> findObstacles(const sensor_msgs::LaserScan& scan,
-                                                        float max_obstacle_angle_diff);
+                                                        float max_obstacle_angle_diff, float max_obstacle_distance_diff);
```

Comment:
just a formatting thing, each new variable should be on a new line, so this should be more like:
```
float max_obstacle_angle_diff, 
float max_obstacle_distance_diff);
```
rather then:
```
float max_obstacle_angle_diff, float max_obstacle_distance_diff);
```